lib/circuitbox/circuit_breaker.rb

```diff
@@ -25,14 +25,14 @@ class CircuitBreaker
     def initialize(service, options = {})
       @service = service.to_s
       @circuit_options = options
-      @circuit_store   = options.fetch(:cache) { Circuitbox.circuit_store }
+      @circuit_store   = options.fetch(:cache) { Circuitbox.default_circuit_store }
```

Comment:
Worth unifying these names at this point?  `cache` vs `circuit_store`?

Reply:
I'll change that in another pr